SConstruct

```diff
@@ -284,102 +336,80 @@ if env['OS'] == 'Darwin':
-    defaults.cxxFlags = '-std=c++11'
-    defaults.buildPch = True
+    defaults.apply("clang")
     env['pch_flags'] = ['-include-pch', 'src/pch/system.h.gch']
 
 else:
     print("WARNING: Unrecognized C compiler '%s'" % env['CC'])
 
-if env['OS'] in ('Windows', 'Darwin'):
-    defaults.threadFlags = ''
-else:
-    defaults.threadFlags = '-pthread'
+if env['OS'] == 'Windows':
+    defaults.apply("Windows")
+elif env["OS"] == "Darwin":
+    defaults.apply("macOS")
 
 # InstallVersionedLib only fully functional in SCons >= 2.4.0
 # SHLIBVERSION fails with MinGW: http://scons.tigris.org/issues/show_bug.cgi?id=3035
 if (env['toolchain'] == 'mingw'
     or parse_version(SCons.__version__) < parse_version('2.4.0')):
-    defaults.versionedSharedLibrary = False
-else:
-    defaults.versionedSharedLibrary = True
-
-defaults.fsLayout = 'compact' if env['OS'] == 'Windows' else 'standard'
-defaults.env_vars = 'PATH,LD_LIBRARY_PATH,PYTHONPATH'
+    defaults.apply("mingw")
 
-defaults.python_prefix = '$prefix' if env['OS'] != 'Windows' else ''
-
-# Transform lists into strings to keep cantera.conf clean
-for key,value in defaults.__dict__.items():
-    if isinstance(value, (list, tuple)):
-        setattr(defaults, key, ' '.join(value))
+defaults.apply("default")
+defaults.python_cmd = sys.executable
 
 # **************************************
 # *** Read user-configurable options ***
 # **************************************
 
 config_options = [
     PathVariable(
-        'prefix',
-        'Set this to the directory where Cantera should be installed.',
+        "prefix",
+        """Set this to the directory where Cantera should be installed. On Windows
+           systems, '$ProgramFiles' typically refers to "C:\Program Files".""",
         defaults.prefix, PathVariable.PathAccept),
     PathVariable(
         'libdirname',
         """Set this to the directory where Cantera libraries should be installed.
-           Some distributions (for example, Fedora/RHEL) use 'lib64' instead of 'lib' on 64-bit systems
-           or could use some other library directory name instead of 'lib' depends
-           on architecture and profile (for example, Gentoo 'libx32' on x32 profile).
-           If user didn't set 'libdirname' configuration variable set it to default value 'lib'""",
+           Some distributions (for example, Fedora/RHEL) use 'lib64' instead of 'lib'
+           on 64-bit systems or could use some other library directory name instead of
+           'lib' depends on architecture and profile (for example, Gentoo 'libx32' on
+           x32 profile). If the user didn't set the 'libdirname' configuration
+           variable, set it to the default value 'lib'""",
         'lib', PathVariable.PathAccept),
     EnumVariable(
         'python_package',
-        """If you plan to work in Python, then you need the ``full`` Cantera Python
+        """If you plan to work in Python, then you need the 'full' Cantera Python
```

Comment:
I haven't gotten to the option processing script yet (so I don't know if this is handled), but the \`\` will put the `full` into monospace in ReST. I think that should be preferred on the web, although it's obviously not optimal for the CLI.

Reply:
The conversion script handles all the back tick \`\` replacements. For the regular `scons help` output the regular `'` is imho more readable. At the moment, the `regex` parser expressions exclude whitespace for single quotes, but include whitespace for double quotes (which can be tweaked)